src/components/layout/Footer.tsx

```diff
@@ -1,3 +1,4 @@
+import { Logo } from '@/components';
```

Comment:
Undo the footer changes, because there is a separate issue for it #233

Reply:
I don't know about the issue you mentioned, but I have already discussed the changes that I have made with @money8203 and @Deepu178

Reply:
that pr needs to be updated with the latest design changes that we've made to our website, till them this will work